src/main/scala/uk/gov/hmrc/SbtAutoBuildPlugin.scala

```diff
@@ -46,11 +46,7 @@ object SbtAutoBuildPlugin extends AutoPlugin {
     logger.info(s"SbtAutoBuildPlugin - adding build settings")
 
     Seq(
-      // targetJvm declared here means that anyone using the plugin will inherit this by default. It only needs to
-      // be specified by clients if they want to override it
-      // TODO default value should be set by DefaultBuildSettings.scalaSettings
-      DefaultBuildSettings.targetJvm := "jvm-1.8",
```

Comment:
Default value has been moved to `DefaultBuildSettings.scalaSettings` (`sbt-settings`)